Cargo.toml

```diff
@@ -46,13 +46,14 @@ lalrpop = { version = "0.19.6", features = [ "lexer" ] }
 built = { version = "0.5.1", features = [ "git2", "semver" ] }
 
 [dev-dependencies]
-pgx-tests = "0.1.21"
+pgx-tests = { git = "https://github.com/zombodb/pgx", rev = "fa8ece3232df3fb75562a376d04a060488a17176" }
 libc = "0.2.99"
 
 [profile.dev]
 panic = "unwind"
 debug = true
 opt-level = 0
+codegen-units = 1
```

Comment:
lets' not commit this

Reply:
or wait, is this necessary now?  I thought it wasn't.

Reply:
It's not! I missed this!